src/database/__init__.py

```
@@ -0,0 +1,21 @@
+__all__ = [
+    'DatabaseConnection',
+    'get_db_connection',
+    'init_database',
+    'check_database_health',
+    'Case',
+    'CaseDocument',
+    'Extraction',
+    'CaseOperations',
+    'DocumentOperations',
+    'BankruptcyDBOperations'
+]
```

Ruff (0.11.9)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)